src/main/java/com/ezasm/gui/toolbar/SimulatorGUIActions.java

```diff
@@ -131,20 +145,24 @@ static void stop() {
      * Handles if the user requests that the running program be temporarily stopped.
      */
     static void pause() {
+        final var pc = getPC();
```

Comment:
`var`? `getPC()` is of type `long`. Use it accordingly. Also why make a variable here instead of just putting it in the `DiscordActivity.setState()`?